src/domain/app/AppYouthProfileRoute.tsx

```diff
@@ -29,12 +29,9 @@ function AppYouthProfileRoute({
   ...rest
 }: Props) {
   const { t } = useTranslation();
-  const [showNotification, setShowNotification] = useState<boolean>(false);
 
   const { data, loading } = useQuery<HasYouthProfile>(HAS_YOUTH_PROFILE, {
-    onError: () => {
-      setShowNotification(true);
-    },
+    onError: () => toastNotification({}),
```

Comment:
Using an empty object is discouraged.

Reply:
Fixed now. Silly mistake on my part to, forgot to make whole `notificationOptions` optional.